.github/workflows/deploy.yml

```diff
@@ -3,6 +3,10 @@ on:
   release:
     types: [published]
   workflow_dispatch:
+  workflow_run:
+    workflows: ["Testimonial"]
+    types:
+      - completed
```

Comment:
I will remove this as we have GitHub Actions that will take various steps before getting to this stage, for example a release and container needs to be created before deploying